### PR TITLE
IMS 1.1 - Tasks reorg and new features

### PR DESCRIPTION
**Pull request by ghost**
This PR includes many changes:

- Fix virt-v2v-wrapper.py call to `_is_iso_domain`: missing `_` for the private method
- Add configuration of SSH transport: install private keys for VMware hosts and set `.ssh/config`
- Add configuration of `ca-trust` for providers certificates to avoid using insecure connection
- Add creation of ManageIQ conversion host record and tagging of associated resource
- Reorganize tasks: split in separate files, rename files, etc...
- Allow to not configure VDDK if `vddk` transport method is not set for conversion host
- Remove nbdkit-* uninstallation as it also uninstalls vdsm

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by fdupont-redhat** [comment]
Moving back to WIP to split it in smaller chunks. Will be closed once done.

**Comment by fdupont-redhat** [comment]
https://github.com/oVirt/ovirt-ansible-v2v-conversion-host/pull/15

File: tasks/install-transport-vddk-nbdkit-plugin-vddk-rpm.yml [hunk 1]
```diff
@@ -7,14 +11,13 @@
...
   yum:
-    state: present
-    name: "{{ item }}"
-  with_items:
-    - gcc
-    - make
-    - rpm-build
-    - yum-utils
-    - nbdkit-devel
+    state: "{{ v2v_yum_check }}"
```
Comment (nyoxi) [inline tasks/install-transport-vddk-nbdkit-plugin-vddk-rpm.yml:14]: I'm pretty sure this was not using `v2v_yum_check` on purpose. Sadly I cannot remember the reason now.

File: tasks/uninstall-transport-ssh.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+- set_fact:
+    ssh_config_path: "/var/lib/vdsm/.ssh"
+    ssh_config_owner: "vdsm"
+  when:
+    - "v2v_host_type == 'rhv'"
```
Comment (nyoxi) [inline tasks/uninstall-transport-ssh.yml:6]: We should not do that unless we're sure the key was created by install role. It is normal to have SSH key configured on VDSM host for other reasons and we cannot remove it arbitrarily.

**Comment by nyoxi** [comment]
Merging the PR. I will drop the SSH key removal in next commit.

**Pull request merged**